Libraries/StyleSheet/processAspectRatio.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+const invariant = require('invariant');
+
+function processAspectRatio(aspectRatio: number | string): ?number {
+  if (typeof aspectRatio === 'number') {
+    return aspectRatio;
+  }
+
+  const match = new RegExp(
+    /(^\s*[+]?\d+([.]\d+)?\s*$)|(^\s*([+]?\d+([.]\d+)?)\s*\/\s*([+]?\d+([.]\d+)?)\s*$)/,
```

Comment:
Quoting @jacdebug:
>This regex is not easily maintainable also fails on values like auto 4 / 3, can we follow spec and check simple splitting by space will work?
https://css-tricks.com/almanac/properties/a/aspect-ratio/#aa-values
Also this transforms will add up and have some runtime cost, we should do static transforms at build time using babel

Reply:
@cipolleschi and @jacdebug react native does not really support `auto` as a value for aspectRatio, any ideas on how we should handle that? Should we default `auto` to some specific value?

Reply:
`auto` can just do nothing since it's the default. And because there is no concept of a "replaced element" in RN, `auto <ratio>` should probably not be allowed. You can split on `/`, trim white space if needed, and parse the float

Reply:
Updated